tools/upload_to_anaconda_staging.sh

```diff
@@ -13,14 +13,6 @@ upload_wheels() {
     VERSION=$(git describe --tags --abbrev=8)
     popd
 
-    if [ "$OPENBLAS_LIBS_STAGING_UPLOAD_TOKEN" != "" ]; then
-        echo "Uploading OpenBLAS $VERSION to anaconda.org/multibuild-wheels-staging"
-
-        anaconda -t $OPENBLAS_LIBS_STAGING_UPLOAD_TOKEN upload \
-                --no-progress --force -u multibuild-wheels-staging \
-                dist/scipy_openblas*.whl
-
-    fi
```

Comment:
This was leftover from the transition to scientific-python-nightly-wheels

Reply:
No longer needed because releases go straight to PyPI now? Or do release builds go to the nighly bucket (unlike for numpy et al)?

Reply:
Wheels go to https://anaconda.org/scientific-python-nightly-wheels/ (below), this stanza was redundantly uploading to https://anaconda.org/multibuild-wheels-staging as well.

Reply:
Okay, if you're sure. For numpy/scipy/etc. the nightly wheels go to the first link, and releases to the latter. Since from this repo not every change turns into a new release, I had expected the same setup. Are you saying that for a release you download from the nightly bucket (by hand?) and then upload to PyPI?

It's possible I'm missing what happens for a release here, or that it's documented somewhere.

Reply:
I updated the build/release process in the README.md. The release wheels are uploaded manually from scientific-python-nighly-wheels. I think we should migrate numpy and scipy away from multibuild-wheels-staging as well, since the newer scientific-python-nightly-wheels container has a cleanup script to remove old files. We are getting close to the 50GB limit on multibuiild-wheels-staging, as can be seen by those with permissions [here](https://anaconda.org/multibuild-wheels-staging/settings/storage), duplicated below for those without permissions:

Size | Artifact | Count
--|--|--
15.4 GB  | scipy  | 448 files
10.1 GB  | openblas-libs  | 561 files
8.4 GB  | pandas  | 755 files
4.5 GB  | numpy  | 330 files
3.6 GB  | statsmodels  | 393 files
1.3 GB  | dipy  | 178 files
376.2 MB  | scipy-openblas32  | 51 files
318.7 MB  | scipy-openblas64  | 43 files
249.0 MB  | yt  | 24 files
114.1 MB  | biopython  | 53 files
97.5 MB  | pywavelets  | 24 files
73.5 MB  | astropy  | 12 files
44.0 MB  | kiwisolver  | 120 files

Some of those files are old, do we really need the 1 year old scipy 1.9 wheels or the numpy 1.24.4 wheels? I removed any openblas-libs befor version 0.20, which saved about half its storage total.

Reply:
> I updated the build/release process in the README.md. 

thanks!

> Some of those files are old, do we really need the 1 year old scipy 1.9 wheels or the numpy 1.24.4 wheels? I removed any openblas-libs befor version 0.20, which saved about half its storage total.

Problem solved by cleaning up `scipy`, took all of 3 minutes. Total storage now: 26.8 GB of 50.0 GB

That's over several years of usage, so doing nothing is fine for a long time again.